Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -10,16 +10,60 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Check for expected values and outcomes in tests.
+Check for expected values, outcomes, and events in tests.
 
 ## Overview
 
-The testing library provides `#expect()` and `#require()` macros you use to 
-validate expected outcomes. To validate that an error is thrown, or _not_ thrown, 
+Use `#expect()` and `#require()` macros to validate expected outcomes.
```

Comment:
Please use double-backticks and link to the `expect(_:_:)` macro rather than writing `#expect()`. Ibid for `#require()`, but note the overloads are different. (This is an error in the existing documentation.)

Reply:
On first use only, or every time?

Reply:
Every time: DocC does not render macros with leading `#` or `@` characters, so having them suddenly appear is likely to be confusing. If, later, DocC does add those characters, we'll get them everywhere "for free".

Reply:
However, if your department already has guidelines for this sort of thing, I defer to you.

Reply:
FWIW, what a developer _types_ is `#expect()` or `#require()`, so I think it's more helpful to show them that; nonetheless, I'll change this to match the local style.

Reply:
We'd much prefer the DocC compiler would add the `#` for us in this case. There's an issue or radar or some such floating around out there about it.